### DISCUSSION TITLE
write_back alternative?

### DISCUSSION THREAD

**Discussion by hainesmatt** [msiemens/tinydb#386] [Q&A]
I just upgraded and saw that "write_back" has been deprecated. I use this a fair amount in my code, so what's the best (simplest) way of achieving the same thing? Thanks!

**Comment by msiemens** [comment]
The easiest way would be to use a custom `Table` class that just adds the removed `write_back` method. The code can be found here: https://github.com/msiemens/tinydb/blob/88d6304bef5bcc4fdbc78a5c2747c407e7e80412/tinydb/database.py#L494-L525

**Reply by hainesmatt** [reply]
Thank you!
